fft/src/KokkosFFT_SYCL_plans.hpp

```diff
@@ -7,6 +7,11 @@
 
 #include <numeric>
 #include <algorithm>
+#if defined(__INTEL_LLVM_COMPILER) && INTEL_MKL_VERSION >= 20250100
```

Comment:
Why are we using `defined(__INTEL_LLVM_COMPILER)` and `defined(INTEL_MKL_VERSION)` ?